### PR TITLE
feat(InputfileResolver): exclude online files from globbing

### PR DESCRIPTION
**Pull request by Kattoor**
This pull request fixed issue #90

### CODE REVIEW

File: src/InputFileResolver.ts [hunk 1]
```diff
@@ -7,13 +7,18 @@ const log = log4js.getLogger('InputFileResolver');
 
 const DEFAULT_INPUT_FILE_PROPERTIES = { mutated: false, included: true };
 
+function isWebUrl(pattern: string): boolean {
+  return pattern.indexOf('http://') === 0 || pattern.indexOf('https://') === 0;
```
Comment (simondel) [inline src/InputFileResolver.ts]: You could also use `pattern.startsWith('http://')` 👍

File: src/InputFileResolver.ts [hunk 2]
```diff
@@ -33,11 +38,26 @@ export default class InputFileResolver {
...
         }
       }
     });
   }
 
+  private validateMutationArray(mutationArray: Array<string>) {
+    if (mutationArray) {
+      mutationArray.forEach(mutation => {
+        if (isWebUrl(mutation)) {
+          throw new Error(`Cannot mutate web url "${mutation}".`)
```
Comment (simondel) [inline src/InputFileResolver.ts]: You're missing a semicolon here: https://www.bithound.io/github/stryker-mutator/stryker/blob/869ee89d6fec2bf3ea98d36491e4efc7aaec8658/src/InputFileResolver.ts

File: src/InputFileResolver.ts (hunk 1 above)
Comment (simondel) [inline src/InputFileResolver.ts]: Turns out this is a part of ES6, thanks for the heads-up @Kattoor

**Comment by simondel** [comment]
This may not work when we copy the files to the sandbox because the file is not on the file system. It may be smart to move `isWebUrl` to a util class so we can use it multiple times.

**Comment by simondel** [comment]
I'll fix this

File: src/Sandbox.ts [hunk 3]
```diff
@@ -64,6 +65,11 @@ export default class Sandbox {
   }
 
   private copyFile(file: InputFile): Promise<void> {
+    if (isOnlineFile(file.path)) {
+      this.fileMap[file.path] = file.path;
+      return Promise.resolve();
+    }
```
Comment (nicojs) [inline src/Sandbox.ts]: Could you add an else here? That is a bit cleaner and safer IMHO.

**Pull request merged**